GPSCoord[] boundaryPoints
DynamicNoFlyZone[] dynamicNoFlyZones
NoFlyZone[] noFlyZones
Road[] roads
GPSCoord startPoint
GPSVelocity startVelocity
GPSCoord[] wayPoints